src/packages/cli/src/__tests__/__snapshots__/studio.test.ts.snap

```diff
@@ -36,7 +36,6 @@ Object {
             string: true,
           },
         },
-        clientMethod: with_all_field_types.create,
```

Comment:
Not sure why this is gone? 🤔

Reply:
Don't think it was used anywhere. [Related](https://github.com/prisma/prisma/pull/6969#issuecomment-848052935)